contracts/DLCManager.sol

```diff
@@ -734,4 +759,24 @@ contract DLCManager is
         dlcBTCPoRFeed = feed;
         emit SetDlcBTCPoRFeed(feed);
     }
+
+    function setBtcMintFeeRateForAddress(
+        address user,
+        uint256 newBtcMintFeeRate
+    ) external onlyAdmin {
+        if (newBtcMintFeeRate > 10000)
```

Comment:
make this a const maybe?

Reply:
or maybe that makes it harder to update later, given how upgradable contracts work?

Reply:
I think that should come with the MAX/MIN fee rates update. This code will change with that PR.